website/content/en/preview/getting-started/getting-started-with-terraform/_index.md

```diff
@@ -153,14 +168,12 @@ module "eks" {
         # Required by Karpenter
         "arn:${local.partition}:iam::aws:policy/AmazonSSMManagedInstanceCore"
       ]
-    }
-  }
 
-  tags = {
-    # Tag node group resources for Karpenter auto-discovery
-    # NOTE - if creating multiple security groups with this module, only tag the
-    # security group that Karpenter should utilize with the following tag
-    "karpenter.sh/discovery" = local.cluster_name
+      tags = {
+        # This will tag the launch template created for use by Karpenter
+        "karpenter.sh/discovery/${local.cluster_name}" = local.cluster_name
```

Comment:
AFAIK, we only use the name of LTs to discover LTs [here](https://github.com/aws/karpenter/blob/5f5ff3a3bfc84f2f23e3ae8e49137ba35b0b2a44/pkg/cloudprovider/aws/launchtemplate.go#L137-L139) and to create [here](https://github.com/aws/karpenter/blob/main/pkg/cloudprovider/aws/apis/v1alpha1/provider.go#L57). I don't understand where this LT discovery tag is being used, can you help me understand here?